### PR TITLE
Fixes in the new commit

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Fixes errors in the new commits.

### CODE REVIEW

File: data/successor 1 introduction.txt [hunk 1]
```diff
@@ -565,7 +565,7 @@ mission "Successors: First Contact 3"
 					goto betterlatethannever
 			action
 				set "Successors FC3: captured kijra-pet"
-			`	You rummage around in the other cargo containers until you find a bit of spare fabric scraps and leaf litter, a couple of handfuls of which you add to the animal's cage. With that sorted, you make your preparations for takeoff.`
+			`	You rummage around in the other cargo containers until you find a bit of spare fabric scraps and leaf litter, a couple of handfuls of which you add to the animal's cage. With that sorted out, you make your preparations for takeoff.`
```
Comment (TheGiraffe3) [inline data/successor 1 introduction.txt:568]: Should this be changed for the other ones as well?

File: data/successor 1 introduction.txt [hunk 2]
```diff
@@ -565,12 +565,12 @@ mission "Successors: First Contact 3"
...
 				goto end
 			label takeoff
 			action
 				set "Successors FC3: captured kijra-pet"
-			`	With the animal sorted, you make your preparations for takeoff.`
+			`	With the animal not having too many problems, you make your preparations for takeoff.`
```
Comment (TheGiraffe3) [inline data/successor 1 introduction.txt]: @Daeridanii1 how are you wanting this to be? This is the best wording I can think of right now, but it's not amazing.

Comment (Daeridanii1) [inline data/successor 1 introduction.txt]:
```suggestion
			`	With the animal stowed away, you make your preparations for takeoff.`
```
Your suggestion is a little wordy for me, but I agree that something more than "sorted" is needed. Does this phrasing work for you?

File: data/map planets.txt [hunk 3]
```diff
@@ -211,7 +211,7 @@ planet Myiara-Aret-Iir
...
-	description `Once a minor colony world, what would come to be known as Myiara-Aret-Iir was the last stop of all those hoping to escape the decay of the Predecessors' space by traveling through the wormhole. Those with wealthy patrons who could afford passage on specially-modified vessels spent little time here, but those without often found the blossoming shanty towns of the planet to be their new homes. With the denounment of the Predecessors' last inter-Houses war and the final push to evacuate, most of those living here eventually made it through. The rest probably perished long ago.`
+	description `Once a minor colony world, what would come to be known as Myiara-Aret-Iir was the last stop of all those hoping to escape the decay of the Predecessors' space by traveling through the wormhole. Those with wealthy patrons who could afford passage on specially-modified vessels spent little time here, but those without often found the blossoming shanty towns of the planet to be their new homes. With the denouncement of the Predecessors' last inter-Houses war and the final push to evacuate, most of those living here eventually made it through. The rest probably perished long ago.`
```
Comment (Daeridanii1) [inline data/map planets.txt]:
```suggestion
	description `Once a minor colony world, what would come to be known as Myiara-Aret-Iir was the last stop of all those hoping to escape the decay of the Predecessors' space by traveling through the wormhole. Those with wealthy patrons who could afford passage on specially-modified vessels spent little time here, but those without often found the blossoming shanty towns of the planet to be their new homes. With the denouement of the Predecessors' last inter-Houses war and the final push to evacuate, most of those living here eventually made it through. The rest probably perished long ago.`
```
'Twas a typo in its original form. "Denouement" is perhaps an overly-articulate word for this (the slow and drawn-out end to a long conflict), but many years of creative writing classes have taught me to be precise---I'll think on this to see if there's a different phrasing I prefer.

File: data/successor 1 introduction.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/successor 1 introduction.txt]: My suggestion was pretty bad, I'll just say.
Yours is great.

File: data/map planets.txt (hunk 3 above)
Comment (TheGiraffe3) [reply data/map planets.txt]: Just thought that the way it was was incorrect. Good to know what it was supposed to be.

**Review by Daeridanii1 — APPROVED** [review]
Looks good. Thanks!

**Comment by TheGiraffe3** [comment]
Always happy to help!

**Pull request merged**